src/main/java/labs/introtoprogramming/lab5/object/Parser/ParseObj.java

```diff
@@ -0,0 +1,487 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import labs.introtoprogramming.lab5.object.Builder.BuilderInterface;
+
+import java.io.*;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.*;
+
+public class ParseObj {
+    private Logger log = Logger.getLogger(ParseObj.class.getName());
+
+    /**
+     * string indicators of parameter in .obj file
```

Comment:
Write comment starting with uppercase letter and don't forget dot at the end.

Reply:
Done:)